src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springboot.datajpa.repository;
+
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
this is redundant
```suggestion
```

Reply:
answering your question why - I think it just looks up the implementations of JpaRepository and finds our based on that